Dfc.CourseDirectory/Resources/ArmTemplates/test-parameters.json

```diff
@@ -14,6 +14,12 @@
     "postcodeApiKey": {
       "value": "not-a-real-key"
     },
+    "getAddressApiUrl": {
+      "value": "https://my.api.net/getaddress"
```

Comment:
`api.net` is a registered domain https://www.whois.com/whois/api.net I suggest sticking to reserved domains (and their subdomains of course) for tests such as `example.org`
https://www.iana.org/domains/reserved

Reply:
I didn't create the file or the convention so this isn't really helpful in the context of this review.

Reply:
I didn't create the convention that already exists for this file. Please feel free to update it.